app/components/EntryProperties.tsx

```diff
@@ -89,14 +87,13 @@ const styles: any = (theme: any) => ({
     overflowY: AppConfig.isFirefox ? 'auto' : 'overlay',
     overflowX: 'hidden',
     flexGrow: 1,
-    // padding: 10,
+    padding: 7,
```

Comment:
With this padding the bottom item from properties (for example map) overlap the bottom window and resize line